packages/cli/src/Db.ts

```diff
@@ -191,6 +192,10 @@ export async function init(testConnectionOptions?: ConnectionOptions): Promise<v
 	collections.Settings = Container.get(SettingsRepository);
 	collections.Credentials = Container.get(CredentialsRepository);
 	collections.Workflow = Container.get(WorkflowRepository);
+
+	const pruningService = Container.get(PruningService);
+
+	if (await pruningService.isPruningEnabled()) pruningService.startPruning();
```

Comment:
though this probably works, might it not be confusing to have the starting of this service 'hidden' away in the Db.ts?

Reply:
So my reasoning was that pruning was already hiden away in `ExecutionRepository` so now that they're separate their inits should remain as close as possible to avoid unexpected side effects.